src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerFormAssetWallet.js

```diff
@@ -53,25 +50,20 @@ export const TriggerFormAssetWallet = ({
 
       <div className={styles.Field}>
         <Label className={styles.label}>&nbsp;</Label>
-        <FormikSelect
-          name='target'
-          isDisabled={defaultAsset.isDisabled}
-          defaultValue={defaultAsset.value.value}
-          placeholder='Pick an asset'
-          options={allProjects.map(asset => ({
-            label: asset.slug,
-            value: asset.slug
-          }))}
-          onChange={newAsset => {
-            if (ethAddress) {
-              if (metaFormSettings.target.value.value === newAsset.value) {
-                setFieldValue('ethAddress', ethAddress)
-              } else {
-                setFieldValue('ethAddress', '')
-              }
-            }
-          }}
-        />
+        {isEthWallet && (
```

Comment:
You can choose component and then use it.
const ProjectsView = isEthWallet ? ProjectByErc20 : ...

<ProjectsView

Reply:
Correct, will fix